lib/_stream_readable.js

```diff
@@ -758,8 +758,12 @@ Readable.prototype.unpipe = function(dest) {
     state.pipesCount = 0;
     state.flowing = false;
 
-    for (var i = 0; i < len; i++)
+    // use the above assignment
+    dests[0].emit('unpipe', this, unpipeInfo);
```

Comment:
What I meant in my previous comment was:

```js
for (var i = 0; i < len; i++)
  dests[i].emit('unpipe', this, { hasUnpiped: false });
```

Reply:
ok~